Traceback (most recent call last):
  File "nmt.py", line 48, in <module>
    from docopt import docopt
ModuleNotFoundError: No module named 'docopt'
